sdk/dnsresolver/Azure.ResourceManager.DnsResolver/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.2 (Unreleased)
+
+### Features Added
+
+### Breaking Changes
+
+### Bugs Fixed
+
+### Other Changes
+
+## 1.0.0-beta.1 (2022-03-03)
```

Comment:
Should we change the date here?

Reply:
No, this package has been released from a feature branch.